app/models/metasploit/cache/license.rb

```diff
@@ -0,0 +1,63 @@
+# Represents licenses like BSD, MIT, etc used to provide license information for {Metasploit::Cache::Module::Instance modules}
+class Metasploit::Cache::License < ActiveRecord::Base
+  extend ActiveSupport::Autoload
```

Comment:
`extend ActiveSupport::Autoload` is only needed if this class would function as a namespace and `autoload <symbol>` were used.  It can be removed since that is not the case currently.